### PR TITLE
Changing gateway to use redirect downloads

### PR DESCRIPTION
**Pull request by acron0**
We now no longer use commands and events to handle downloads; we hit an endpoint on the gateway which 302's.

https://www.pivotaltracker.com/story/show/138178375

### CODE REVIEW

**Review by tcoupland — APPROVED** [review]

File: src/cljs/witan/ui/controllers/datastore.cljs [hunk 1]
```diff
@@ -127,7 +127,7 @@
   (if (:error data)
     (let [id (first (get-in data [:original :params]))
           tries (data/get-in-app-state :app/datastore :ds/query-tries)]
-      (if (< tries 3)
+      (if (< tries 5)
```
Comment (otfrom) [inline src/cljs/witan/ui/controllers/datastore.cljs:130]: why the change to 5?

**Review by otfrom — COMMENTED** [review]
Just a question. :shipit:

File: src/cljs/witan/ui/controllers/datastore.cljs (hunk 1 above)
Comment (acron0) [reply src/cljs/witan/ui/controllers/datastore.cljs:130]: This related to https://www.pivotaltracker.com/story/show/138820445 - it's a patch which hopefully means no more '401' visible issues
